scspell/__init__.py

```diff
@@ -670,8 +670,8 @@ def find_dict_file(override_dictionary):
     return os.path.expandvars(os.path.expanduser(dict_file))
 
 
-def spell_check(source_filenames, override_dictionary=None,
-                base_dicts=[],
+def spell_check(source_filenames, ext=[''],
```

Comment:
add new arguments at the end, otherwise if anyone is calling `spell_check('blah', ['override','dict'], ['base','dict']..)` then you will have broken their program.